README.md

```diff
@@ -0,0 +1,242 @@
+# Introduction
+
+This library provides easy to use interface of the Bolt Cloud APIs. The functions are similar in syntax to standard Arduino or Rasberry Pi functions.
+
+# Installation
```

Comment:
I think this is a good way for installing the go package. 

Installing
go get
$ go get -u github.com/Inventrom/bolt-api-go
Example
Getting all groups
import (
	"fmt"

	"github.com/Inventrom/bolt-api-go"
)

Reply:
I will implement that.